src/batou_scm/tests/test_source.py

```diff
@@ -24,21 +23,24 @@ def test_all_clones_are_configured_as_subcomponents(source):
     clones = [s for s in source.sub_components if isinstance(s, Clone)]
     assert set(clones) == set(source.clones.values())
     assert set(clone.url for clone in clones) == set([
-        'https://example.com/foo',
-        'https://example.com/bar',
-        'https://example.com/baz'])
+        'https://example.com/foo', 'https://example.com/bar',
+        'https://example.com/baz'
+    ])
```

Comment:
I do not think this change is necessary, it does not improve readability.

Reply:
Weird how that happened. Maybe 2to3, maybe something else ...